src/ContactPeople/ContactPeopleViewContainer.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { withRouter } from 'react-router-dom';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Icon } from '@folio/stripes/components';
+import {
+  batchFetch,
+  useShowToast,
+} from '@folio/stripes-acq-components';
+
+import { baseContactsResource } from '../common/resources';
+import ContactPeopleView from './ContactPeopleView';
+
+const ContactPeopleViewContainer = ({
+  contactsIds,
+  history,
+  match,
+  mutator,
+  vendorCategories,
+}) => {
+  const organizationId = match.params.id;
+  const [contacts, setContacts] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const showToast = useShowToast();
+
+  useEffect(() => {
+    setContacts([]);
+    if (contactsIds.length) {
+      setIsLoading(true);
+      batchFetch(mutator.orgContacts, contactsIds, (itemsChunk) => {
+        const contactsQuery = itemsChunk
```

Comment:
basically, it looks like a quite general callback - to filter items by id. Maybe it's better to re-use this instead of copy-pasting

Reply:
Added common `buildQuery` callback - [stripes-acq-components](https://github.com/folio-org/stripes-acq-components/pull/131)